core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java

```diff
@@ -99,13 +99,16 @@ public void passivateAll() throws PersistenceException {
       if (enabled && !skipOnStop) {
          long start = timeService.time();
          log.passivatingAllEntries();
+         int count = 0;
          for (InternalCacheEntry e : container) {
+            if (count != Integer.MAX_VALUE) {
+               count++;
```

Comment:
That is what I had originally, but the passivatedEntries log method requires int. Want me to update both?